tests/autoyast/installation.pm

```diff
@@ -103,6 +116,10 @@ sub run {
                 wait_idle(5);
                 $num_errors++;
             }
+            if (match_has_tag('linuxrc-install-fail')) {
+                save_logs_in_linuxrc("stage1_error$i");
+                die "installation ends in linuxrc";
```

Comment:
@nadvornik I am fine if you say this does not make sense to you and we continue as you proposed :-)